### DISCUSSION TITLE
Problem while using mirage layout

### DISCUSSION THREAD

**Discussion by rishityagi2019** [primefaces/community#1514] [PrimeFaces Templates] [answered]
Hi, I purchased Mirage layout from the primefaces some time back, also downloaded the zip file provided after making payments . Now i am keen to make use of this layout inside a jsf project using jakarta ee 10 but not able to understand how can we use the content given in the zip file as it seems quite confusive also there is no implementation/usage guide provided with the layout.

**Comment by melloware** [comment, marked as answer]
@rishityagi2019 did you run `mvn clean jetty:run` on the latest downloaded mirage and then navigate to `https://localhost:8080/mirage/documentation.xhtml`  The documentation should be there and look something like this: 
https://www.primefaces.org/freya/documentation.xhtml

**Comment by tandraschko** [comment]
new releases contains a readme as well

also i will create a docu how to integrate it in a existing project too
